backend/dashboard-server.js

```diff
@@ -0,0 +1,29 @@
+import { get, update } from '@binaris/shift-db';
+
+/*
+  Generate random chart data
+ */
+/* @expose */
+export async function generateRemoteData() {
+  return [
+    Math.floor(Math.random() * 100 + 1),
+    Math.floor(Math.random() * 100 + 1),
+    Math.floor(Math.random() * 100 + 1),
+  ];
+}
+
+/*
+  Save chart data in db
+ */
+/* @expose */
+export async function setStorageData(data) {
+  return await update('chartData', (prevData = []) => data);
```

Comment:
In this use case it might be better to just create the initial data without the option of updating.
```suggestion
  return await create('chartData', data);
```

Reply:
Also add the missing import

Reply:
No need to await if you're returning the value.

Reply:
Done, Thanks.